scripts/configure-ironic.sh

```diff
@@ -52,7 +52,11 @@ export IRONIC_IPA_COLLECTORS=${IRONIC_IPA_COLLECTORS:-default,logs}
 wait_for_interface_or_ip
 
 # Hostname to use for the current conductor instance.
-export IRONIC_CONDUCTOR_HOST=${IRONIC_CONDUCTOR_HOST:-${IRONIC_URL_HOST}}
+if [[ "${SKIP_PROVISIONING_INTERFACE:-false}" == "true" ]]; then
+    export IRONIC_CONDUCTOR_HOST=${IRONIC_CONDUCTOR_HOST:-${IRONIC_URL_HOST:-localhost}}
+else
+    export IRONIC_CONDUCTOR_HOST=${IRONIC_CONDUCTOR_HOST:-${IRONIC_URL_HOST}}
+fi
```

Comment:
```suggestion
export SKIP_PROVISIONING_INTERFACE=${SKIP_PROVISIONING_INTERFACE:-false}
export IRONIC_CONDUCTOR_HOST=${IRONIC_CONDUCTOR_HOST:-${IRONIC_URL_HOST}}
if [[ "${SKIP_PROVISIONING_INTERFACE}" == "true" ]]; then
    export IRONIC_CONDUCTOR_HOST=${IRONIC_CONDUCTOR_HOST:-localhost}}
fi
```
Something like this would be easier to read and since ironic-common.sh is sourced into other scripts, avoids the `:-false` part on each if.

Reply:
had to clean it up a bit more make it work but PTAL thanks!